rest/src/commonMain/kotlin/service/EntitlementService.kt

```diff
@@ -0,0 +1,57 @@
+package dev.kord.rest.service
+
+import dev.kord.common.entity.DiscordEntitlement
+import dev.kord.common.entity.Snowflake
+import dev.kord.rest.json.request.TestEntitlementCreateRequest
+import dev.kord.rest.request.RequestHandler
+import dev.kord.rest.route.Position
+import dev.kord.rest.route.Route
+
+public class EntitlementService(handler: RequestHandler) : RestService(handler) {
```

Comment:
```suggestion
public class EntitlementService(requestHandler: RequestHandler) : RestService(requestHandler) {
```
the param is named `requestHandler` in all services except `ApplicationService`